### ISSUE TITLE
Cannot open different sessions

### ISSUE THREAD

**Issue by akasaka99** [mozilla/multi-account-containers#1472]
- Multi-Account Containers Version:
- Operating System + Version: Mac 10.13.6
- Firefox Version: 68.0.1
- Other installed Add-ons + Version + Enabled/Disabled-Status:
Amazon.com	1.1	true	amazondotcom@search.mozilla.org
AudioContext Fingerprint Defender	0.1.3	true	{ab0ae774-f22f-479b-9b1b-6aff11bf6f5c}
Bing	1.0	true	bing@search.mozilla.org
Blur	8.1.2491	true	donottrackplus@abine.com
CanvasBlocker	0.5.13	true	CanvasBlocker@kkapsner.de
Cookie AutoDelete	3.0.2	true	CookieAutoDelete@kennydo.com
Dark Mode	0.3.3	true	{174b2d58-b983-4501-ab4b-07e71203cb43}
Decentraleyes	2.0.12	true	jid1-BoFifL9Vbdl2zQ@jetpack
Disable WebRTC	1.0.21	true	jid1-5Fs7iTLscUaZBgwr@jetpack
Disconnect for Facebook™	0.1.6	true	{02bf35c3-6f86-4eb4-bea8-e70bc294a7dc}
DuckDuckGo	1.0	true	ddg@search.mozilla.org
Facebook Container	2.0.1	true	@contain-facebook
Firefox Multi-Account Containers	6.1.0	true	@testpilot-containers
FoxyProxy Standard	6.6.2	true	foxyproxy@eric.h.jung
Google	1.0	true	google@search.mozilla.org
Google Container	1.4.0	true	@contain-google
google-no-tracking-url	3.0.0	true	jid1-zUrvDCat3xoDSQ@jetpack
HTTPS Everywhere	2019.6.27	true	https-everywhere@eff.org
LastPass: Free Password Manager	4.29.0.4	true	support@lastpass.com
No Coin - Block miners on the web!	0.4.13	true	{5657c026-efc3-4860-b43b-16e4eaa8a9aa}
OneTab	1.29	true	extension@one-tab.com
Privacy Badger	2019.7.1.1	true	jid1-MnnxcxisBPnSXQ@jetpack
Referer Control	1.31	true	{cde47992-8aa7-4206-9e98-680a2d20f798}
Turn Off the Lights for YouTube™	4.0.41.0	true	stefanvandamme@stefanvd.net
Twitter	1.0	true	twitter@search.mozilla.org
uBlock Origin	1.21.2	true	uBlock0@raymondhill.net
Video DownloadHelper	7.3.7	true	{b9db16a4-6edc-47ec-a1f4-b86292ed211d}
Wikipedia (en)	1.0	true	wikipedia@search.mozilla.org
Privacy Settings	0.3.7	false	jid1-CKHySAadH4nL6Q@jetpack
Wikiwand: Wikipedia Modernized

### Steps to reproduce
1.
2.
3.

**Comment by kendallcorner** [comment]
I can't reproduce this. Is it possible you logged into both containers? You may want to delete the containers, clear cookies and try again. If you do this, please provide steps to reproduce. Thanks!

**Comment by gkaklas** [comment]
> I seems the cookies are shared betwen my Gmail Containers??

Yes, every tab opened in a container (e.g. "Gmail") share their cookies. I believe that to achieve what you are trying to do, you should try creating two containers, e.g. "Gmail1", "Gmail2"; after that, tabs that you open in the container "Gmail1" will have a different set of cookies than the tabs opened in "Gmail2", and will be able to stay logged in in different accounts :thinking:
